### ISSUE TITLE
Tap displayname in timeline to mention

### ISSUE THREAD

**Issue by ara4n** [element-hq/element-x-ios#1527]
A nice feature of classic Element is that if you tap on the displayname of a user in the timeline, it adds a mention to them into the composer. I miss it.

**Comment by ShadowJonathan** [comment]
IIRC this has been decided upon by design/product in element-meta somewhere to be standardised across apps in some fashion

**Comment by ShadowJonathan** [comment]
Found it, with a sign-off: https://github.com/vector-im/element-meta/issues/1701

And it got resolved for EX here: https://github.com/vector-im/element-x-ios/issues/1017

**Comment by pixlwave** [comment]
Thanks for pointing it out 👍

**Comment by ara4n** [comment]
I *really* miss this. Right now tapping on the displayname does nothing and is just a wasted opp, whereas i almost never use the fiddly mentions UI (hit shift, then @, then type the name... rather than simply tap once on the name you wish to mention).

@amshakal, please can we reconsider adding this, if only as a massive accelerator for powerusers?

**Comment by ShadowJonathan** [comment]
As a poweruser, I'd rather see a tab-seperation between DMs and other rooms, or Spaces implementation, before this feature.

**Comment by ara4n** [comment]
I continue to be bitten massively by this. I think the key thing is that most of the time when you want to mention someone, it's because they're hanging around the conversation, and so the convenience of being able to tap on their name to @-mention them is super convenient as opposed to the multiple "shift, at, start typing, wait for autocomple debounce timer, tap to select" 5 or more tap process.

**Comment by simonmichael** [comment]
Yes! Please reconsider, UX team. Tapping a nick to begin a message addressed to that person is a massive time saver. Tapping the icon to view their details still works fine as in Element web/desktop.

**Comment by bmarty** [comment]
In case your missed it @ara4n , the design decision to not implement "mention on user name click" is here: https://github.com/element-hq/element-meta/issues/1701#issuecomment-1568115535

(Seeing this now when doing some cleanup on EXA PRs, in particular this one: https://github.com/element-hq/element-x-android/pull/3858)

**Comment by ara4n** [comment]
we now have a pretty bottompanel which appears when you tap on someone's face or displayname. we could solve this by adding a "mention user" option to that panel, so we can insert an `@mention` easily from there.

**Comment by amshakal** [comment]
Since we have this bottom sheet, we can potentially use this for a bunch of user related features in the room. See example (now in liquid glass). [Link to figma](https://www.figma.com/design/ux3tYoZV9WghC7hHT9Fhk0/Compound-iOS-Components?node-id=4404-38160&t=I3KXvklDaF7RJaWu-1)

<img width="414" height="656" alt="Image" src="https://github.com/user-attachments/assets/b3f81f0e-ce76-4be8-a425-ba9f83aae064" />

**Comment by ara4n** [comment]
it would be AMAZING to have the role configuration in here too - i was literally about to file a bug for the fact that whenever i want to promote/demote someone in a room, i go to their member info... and then get lost. And then remember i have to look in Role & Permissions instead.
